tools/build_go_exe.sh

```diff
@@ -71,6 +71,7 @@ GOPATH_REPO_ORG=${ORG_PATH:=github.com/mesosphere}
 GOPATH_REPO_ORG_DIR=${GOPATH}/src/${GOPATH_REPO_ORG}
 # ex: /.gopath/src/github.com/mesosphere/dcos-commons/sdk/cli
 GOPATH_EXE_DIR="$GOPATH_REPO_ORG_DIR/$REPO_NAME/$RELATIVE_EXE_DIR"
+DEFAULT_GO_ARCH=${DEFAULT_GO_ARCH:-"amd64"}
```

Comment:
I suggest shortening to `GO_ARCH=` as the default is already implied when you set it to `amd64`. That way, if users want to use a different version, it won't be as awkward when they do something like 
```
DEFAULT_GO_ARCH=386 ./build.sh ...
```